generators/app/templates/_dockerTaskGeneric.sh

```diff
@@ -4,7 +4,7 @@ isWebProject=<%= isWebProject %>
 
 # Kills all running containers of an image and then removes them.
 cleanAll () {
-    # List all running containers that use $imageName, kill them and then remove them.
+    # List all running containers that use $serviceName, kill them and then remove them.
```

Comment:
That was a mistake, with the opposite mistake in the .sh file :/

---
In reply to: [66712237](https://github.com/Microsoft/generator-docker/pull/63#discussion_r66712237) [](ancestors = 66712237)